lib/omnitrade_api/client.rb

```diff
@@ -76,5 +74,18 @@ def setup_auth_keys(options)
     def check_auth!
       raise ArgumentError, 'Missing access key and/or secret key' if @auth.nil?
     end
+
+    def raise_error(response)
+      if response.body['error'].nil?
```

Comment:
aqui teria q fazer parse do body tbm ne?

Reply:
se ele retornar um erro http tipo um 404, ele não vem body
se for erro da API, ele retorna